templates/style/rustdoc-common.scss

```diff
@@ -98,6 +98,15 @@ div.rustdoc {
                     top: $top-navbar-height;
                 }
             }
+
+            // A later version of rustdoc uses the .sidebar.shown instead of .sidebar.mobile, and
+            // also needs a height adjustment.
+            &.shown {
+                width: 250px;
+                margin-left: 0;
+                top: 45px + $top-navbar-height !important;
+                height: calc(100vh - 78px) !important;
```

Comment:
Why the `78px`?

Reply:
78px is 45px (rustdoc topbar) plus 33px (docsrs topbar).

Reply:
Then better use `$top-navbar-height` like you did above and add a comment about the rustdoc topbar. Which now brings a question: for older crates, won't your fix break the sidebar?

Reply:
It won't break the sidebar because older rustdoc versions didn't use the `.shown` class.

Reply:
Good point!